packages/core/src/components/ModalNew/layouts/ModalBasicLayout/__stories__/ModalBasicLayout.stories.tsx

```diff
@@ -0,0 +1,468 @@
+import Button from "../../../../Button/Button";
+import { createPortal } from "react-dom";
+import Text from "../../../../Text/Text";
+import Link from "../../../../Link/Link";
+import TransitionView from "../../../../TransitionView/TransitionView";
+import ModalFooterWizard from "../../../footers/ModalFooterWizard/ModalFooterWizard";
+import useWizard from "../../../../../hooks/useWizard/useWizard";
+import { Checkbox } from "../../../../Checkbox";
+import IconButton from "../../../../IconButton/IconButton";
+import { Menu } from "@vibe/icons";
+import { withOpenedModalPreview } from "../../../Modal/__stories__/Modal.stories.helpers";
+
+type Story = StoryObj<typeof Modal>;
+
+const metaSettings = createStoryMetaSettingsDecorator({
+  component: Modal
+});
+
+export default {
+  title: "Internal/Components/Modal [New]/Basic modal",
+  component: Modal,
+  subcomponents: { ModalBasicLayout, ModalHeader, ModalContent, ModalFooter, ModalFooterWizard, TransitionView },
+  argTypes: metaSettings.argTypes,
+  decorators: metaSettings.decorators,
+  tags: ["internal"]
+} satisfies Meta<typeof Modal>;
+
+export const Overview: Story = {
+  decorators: [(Story, context) => withOpenedModalPreview(Story, { isDocsView: context.viewMode === "docs" })],
+  render: (args, { show, setShow }) => {
+    return (
+      <Modal id="modal-basic" show={show} size="medium" onClose={() => setShow(false)} {...args}>
+        <ModalBasicLayout>
+          <ModalHeader
+            title="Modal title"
+            description={
+              <Text type="text1">
+                Modal subtitle, can come with icon <Link inheritFontSize inlineText text="and link." />
+              </Text>
+            }
+          />
+          <ModalContent>
+            <Text type="text1" align="inherit" element="p">
+              Modal content will appear here, you can custom it however you want, according to the user needs. Please
+              make sure that the content is clear for completing the relevant task.
+            </Text>
+          </ModalContent>
+        </ModalBasicLayout>
+        <ModalFooter
+          primaryButton={{ text: "Confirm", onClick: () => setShow(false) }}
+          secondaryButton={{ text: "Cancel", onClick: () => setShow(false) }}
+        />
+      </Modal>
+    );
+  },
+  parameters: {
+    docs: {
+      liveEdit: {
+        isEnabled: false
+      }
+    }
+  }
+};
+
+export const Sizes: Story = {
+  render: () => {
+    const [showSmall, setShowSmall] = useState(false);
+    const [showMedium, setShowMedium] = useState(false);
+    const [showLarge, setShowLarge] = useState(false);
+
+    return (
+      <>
+        <Flex gap="large" style={{ paddingBlock: 40 }}>
+          <Button onClick={() => setShowSmall(true)}>Small</Button>
+          <Button onClick={() => setShowMedium(true)}>Medium</Button>
+          <Button onClick={() => setShowLarge(true)}>Large</Button>
+        </Flex>
+        {createPortal(
```

Comment:
Why aren't we using a portal in the Modal?

Reply:
will add it, created a task